youtube_dl/extractor/generic.py

```diff
@@ -2438,6 +2453,12 @@ def _playlist_from_matches(matches, getter=None, ie=None):
             return _playlist_from_matches(
                 openload_urls, ie=OpenloadIE.ie_key())
 
+        # Look for Visir embeds
+        visir_urls = VisirMediaIE._extract_urls(webpage)
```

Comment:
I don't understand what you mean by that?
These Visir articles can contain self-hosted videos, but it also occurs that youtube or twitter videos are embedded. If a self-hosted video is in the article, the `<iframe ...>` section is generated dynamically, so it cannot be directly extracted from the webpage.

So should these Visir articles be treated in the generic information extractor, or should I set up an own information extractor for Visir articles?